synchronization/src/main/java/de/cyface/synchronization/SyncService.java

```diff
@@ -28,7 +28,7 @@
     /**
      * The synchronisation adapter this service is supposed to call.
      */
-    // TODO Ugh. Singleton is so ugly. Nevertheless this is how it is specified in the documentation. Maybe try to
```

Comment:
I don't think we need to create a task for this so I removed the TODO (because TODOs are only allowed with an annotated Task)